src/Redux_Todos_Thunk/reducers/reducers.js

```diff
@@ -1,32 +1,38 @@
 const initialState = {
   todos: [],
```

Comment:
According to a previous comment
```
const initialState = {
  todos: [],
  page: 1,
  limit: 5,
  total: 0
}
```

Also, actions for handling pagination and limit